.github/workflows/component-bumper.yaml

```diff
@@ -5,26 +5,30 @@ on:
     - cron:  '0 * * * *'
 
 jobs:
-
   build:
-    name: main - CNAO Component Bump Job
+    name: CNAO Component Bump Job
     if: (github.repository == 'kubevirt/cluster-network-addons-operator')
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        branche:
+          - main
+          - release-0.42
+          - release-0.44
+          - release-0.53
+          - release-0.58
     steps:
 
-    - name: Set environment variables
-      run: echo "BASE_BRANCH=main" >> $GITHUB_ENV
-
     - name: Login to Quay
       run: docker login -u="kubevirt+network" -p="${{ secrets.QUAY_ROBOT_TOKEN }}" quay.io
 
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
       with:
-        ref: ${{ env.BASE_BRANCH }}
+        ref: ${{ matrix.branch }}
 
     - name: Pull latest of latest branch
-      run:  git pull --ff-only --rebase origin ${{ env.BASE_BRANCH }}
+      run:  git pull --ff-only --rebase origin ${{ matrix.branch }}
 
     - name: Run bumper script
-      run: make ARGS="-config-path=components.yaml -token=${{ secrets.GITHUB_TOKEN }} -base-branch=${{ env.BASE_BRANCH }}" auto-bumper
+      run: make ARGS="-config-path=components.yaml -token=${{ secrets.GITHUB_TOKEN }} -base-branch=${{ matrix.branch }}" auto-bumper
```

Comment:
Don't we want to use the KUBEVIRT_BOT_TOKEN?

Reply:
Need rebase